### PR TITLE
Downgrade cosmwasm-std and cosmwasm-schema in injective-cosmwasm, injective-math and injective-testing. Add new injective-math

### PR DESCRIPTION
**Pull request by nicolasbaum**

### CODE REVIEW

File: packages/injective-cosmwasm/Cargo.toml [hunk 1]
```diff
@@ -11,22 +11,22 @@ license = "Apache-2.0"
 name = "injective-cosmwasm"
 readme = "README.md"
 repository = "https://github.com/InjectiveLabs/cw-injective/tree/master/packages/bindings"
-version = "0.2.15"
+version = "0.2.16"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cosmwasm-std    = { version = "1.4.1" }
+cosmwasm-std    = { version = "1.2.0" }
```
Comment (maurolacy) [inline packages/injective-cosmwasm/Cargo.toml:18]: Are you sure this is the right version? I mean, there's also a `1.3.x` series.

By the way, the general recommendation is using the latest compatible version in the series (that would be, `1.2.7`) when specifying deps. That eases cargo dependency resolution, and strives towards using the most mature / stable version in the series.

Comment (nicolasbaum) [reply packages/injective-cosmwasm/Cargo.toml:18]: Thanks for the suggestion. Quick question: cosmwasm-schema, cosmwasm-storage and cosmwasm-vm should follow the same versioning as cosmwasm-std correct?

Comment (maurolacy) [reply packages/injective-cosmwasm/Cargo.toml:18]: Yes.

Comment (gorgos) [reply packages/injective-cosmwasm/Cargo.toml:18]: `1.2.7` should be good

**Review by gorgos — CHANGES_REQUESTED** [review]
please change to `1.2.7` as suggested by Mauro

**Pull request closed without merging**